docs/quickstart.ipynb

```diff
@@ -188,11 +188,13 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "\n",
+    "import time\n",
```

Comment:
is there a reason to replace `%timeit` here? Did you have any problems with the measurements?

Reply:
Yes, timeit runs the same function several time, which, in the case of scarlet does not make much sense since it updates some of the objects such that the second run starts as initialise by the result of the previous run. I just wanted to make sure that this was not causing anything weird with regard to the measurement. Also `%timeit` gives results like 80 ms, which is not what we experience. 12s, the result from time.clock is much more realistic.
Actually because %timeit repeats the same command we should not using on blend.fit because the first runtime will alway be MUCH larger than the following ones.

Reply:
Then `%time` is your friend ...